src/Intercom/Intercom.csproj

```diff
@@ -1,24 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
-    <ReleaseVersion>3.0.0</ReleaseVersion>
-    <PackageId>Intercom.Dotnet.Client</PackageId>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>

```

Comment:
We've seen issues when targeting a single framework with the plural `TargetFrameworks`. I believe one targeting a single framework it should be `<TargetFramework>` (singular version).